lib/skate/settings/test_group.ex

```diff
@@ -50,6 +52,13 @@ defmodule Skate.Settings.TestGroup do
     |> Enum.map(&convert_from_db_test_group(&1))
   end
 
+  @spec get_enabled() :: [t()]
+  def get_enabled() do
```

Comment:
**suggestion:** The name `get_enabled` is a little confusing to me because it only includes test groups that are enabled via override, not test groups that are enabled on a per-user basis for a given user (basically, I feel like "enabled" is still a function of both a test group and a user, not just a test group). I think something like `get_override_enabled` would make more sense personally.

Reply:
Done